gnocchi/storage/redis.py

```diff
@@ -111,14 +110,36 @@ def _get_or_create_unaggregated_timeseries(self, metrics, version=3):
         }
         return ts
 
-    def _list_split_keys(self, metric, aggregation, granularity, version=3):
+    def _list_split_keys(self, metric, aggregations, version=3):
```

Comment:
Do you plan to batch also metrics here ? like a dict {metric_id: [aggregations]}

Reply:
Very likely. I'm not there yet, but it'd make sense when we batch metric processing per sack.